packages/toolpad-app/package.json

```diff
@@ -110,7 +110,8 @@
     "superjson": "^1.12.2",
     "ts-node": "^10.9.1",
     "web-streams-polyfill": "^3.2.1",
-    "whatwg-url": "^12.0.0"
+    "whatwg-url": "^12.0.0",
+    "formik": "2.2.9"
```

Comment:
Isn't Formik unmaintained?
I think I even somewhere that the author himself is now recommending using other libraries such as `react-hook-form`.
I always liked Formik though and am most familiar with it compared to other libraries.

Reply:
for the record, we're currenlty already using `react-hook-form` in some places in Toolpad.

Reply:
Thanks, will change the implementation accordingly